features/checkout/shipping_order/seeing_shipping_method_when_all_units_match_to_shipping_category.feature

```diff
@@ -0,0 +1,51 @@
+@checkout
+Feature: Seeing shipping methods which category is same as category of all my units
+    I order to selecting only shipping method proper for my shipment
+    As a Customer
+    I want to be able to choose shipping method which category is same as category of all my units
```

Comment:
... choose **only** shipping methods that match shipping category of all my items